Networking/NetworkingTests/Remote/ProductsRemoteTests.swift

```diff
@@ -125,6 +125,106 @@ final class ProductsRemoteTests: XCTestCase {
         XCTAssertEqual(result?.isFailure, true)
     }
 
+    // MARK: - Delete Product
+
+    func test_deleteProduct_with_success_mock_returns_a_product() {
+        // Given
+        let remote = ProductsRemote(network: network)
+
+        /// When we delete a product, it return the deleted product.
+        network.simulateResponse(requestUrlSuffix: "products/\(sampleProductID)", filename: "product-add")
```

Comment:
nit: if both adding and deleting a product returns the same response, maybe we can rename the `product-add.json` to include the delete case like `product-add-or-delete.json`?

Reply:
It makes sense 👍 . Changed here: 0de6a7a9b2e7060c5b8e7c1ad5cfc2ed07974b9c